lib/targets/cuda/malloc.cpp

```diff
@@ -366,7 +359,7 @@ namespace quda
     }
     track_malloc(MANAGED, a, ptr);
 #ifdef HOST_DEBUG
-    qudaMemset(ptr, 0xff, size);
+    cudaMemset(ptr, 0xff, size);
```

Comment:
Not here specifically, but in the function `shmem_malloc_` below, on line 384, `qudaMemset` -> `cudaMemset`

Reply:
Good catch.  We should probably be using `cudaMemset` here uniformly.  Otherwise a profile.tsv produced from a debug build and devel/release build will differ.

Reply:
3eb6eafeb